api/views/people.py

```diff
@@ -0,0 +1,44 @@
+from django.http import Http404
+from rest_framework import generics, status
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from api.models.people import Student  # , Teacher
+from api.serializers.people import StudentSerializer  # , TeacherSerializer
+
+
+class StudentList(generics.ListCreateAPIView):
+    queryset = Student.objects.all()
+    serializer_class = StudentSerializer
+
+
+class StudentDetail(APIView):
```

Comment:
Please have a look at `ModelViewSet` in [the tutorial](https://www.django-rest-framework.org/tutorial/6-viewsets-and-routers/#refactoring-to-use-viewsets) and [the docs](https://www.django-rest-framework.org/api-guide/viewsets/#modelviewset) of Django REST: it can replace these two classes.